src/raster/r.forcircular/Makefile

```
@@ -0,0 +1,7 @@
+MODULE_TOPDIR = ../..
+
+PGM = r.forcircular
+
+include $(MODULE_TOPDIR)/include/Make/Script.make
+
+default: script
```

```suggestion
default: script
```

Missing newline char added